webapp/app/_components/footer/footer.tsx

```diff
@@ -0,0 +1,113 @@
+              href="https://github.com/srbhr/Resume-Matcher/blob/master/LICENSE"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Apache 2.0
+            </a>
+          </ul>
+        </div>
+        <div className="flex justify-around text-xl w-3/5">
+          {data.map((section, index) => (
+            <div key={index}>
+              <p className="pb-4 font-semibold">{section.label}</p>
+              <ul>
+                {section.items.map((item, itemIndex) => (
+                  <li
+                    className="py-1 text-base hover:text-[#A49DA2]"
+                    key={itemIndex}
+                  >
+                    <a
+                      target={item.shouldOpenInNewPage ? "_blank" : undefined}
+                      rel={
+                        item.shouldOpenInNewPage
+                          ? "noopener noreferrer"
+                          : undefined
+                      }
+                      href={item.link}
+                    >
+                      {item.title}
+                    </a>
+                  </li>
+                ))}
+              </ul>
+            </div>
+          ))}
+        </div>
+      </div>{" "}
+      <div className="flex pt-16">
+        <div className="flex gap-5">
+          <a
+            href="https://discord.com/invite/t3Y9HEuV34"
+            title="Github"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <div className="p-2 rounded-xl hover:p-2 hover:bg-[#44365F] transition duration-300">
+              <Image
+                className="hover:bg-[#44365F]"
+                src="/icons/discord-light.svg"
+                width={20}
+                height={20}
+                alt="GitHub Octocat Logo"
+              />{" "}
+            </div>
+          </a>
+
+          <a
+            href="https://www.producthunt.com/posts/resume-matcher"
+            title="Discord"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <div className="p-2 rounded-xl hover:p-2 hover:bg-[#44365F] transition duration-300">
+              <Image
+                className="hover:bg-[#44365F]"
+                src="/icons/productHunt-light.svg"
+                width={20}
+                height={20}
+                alt="GitHub Octocat Logo"
+              />{" "}
+            </div>
+          </a>
+          <a
+            href="https://github.com/srbhr/Resume-Matcher"
+            title="Github"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <div className="p-2 rounded-xl hover:p-2 hover:bg-[#44365F] transition duration-300">
+              <Image
+                className="hover:bg-[#44365F]"
+                src="/icons/github-light.svg"
+                width={20}
+                height={20}
+                alt="GitHub Octocat Logo"
+              />
+            </div>
+          </a>
```

Comment:
1. In a previous [comment](https://github.com/srbhr/Resume-Matcher/pull/212#discussion_r1375309401), I suggested that the `<div>` wrapper was redundant, as we can simply apply the styles on the `<a>` instead.

As a bonus this would cut down the size of the rendered DOM tree,  and keep the DOM (and React component) lean.

```suggestion
          <a
            className="p-2 rounded-xl hover:p-2 hover:bg-[#44365F] transition duration-300"
            href="https://github.com/srbhr/Resume-Matcher"
            title="Github"
            target="_blank"
            rel="noopener noreferrer"
          >
            <Image
              className="hover:bg-[#44365F]"
              src="/icons/github-light.svg"
              width={20}
              height={20}
              alt="GitHub Octocat Logo"
            />
          </a>
```